### ISSUE TITLE
about uncertainties

### ISSUE THREAD

**Issue by ShellingFord221** [kumar-shridhar/PyTorch-BayesianCNN#15]
Hi, are there any possibilities to compute aleatoric uncertainty and epistemic uncertainty of each class of the model (BCNN), other than just one sample?

**Comment by kumar-shridhar** [comment]
Hi. 
Yes, it is possible. I will put the updated code on this [repo](https://github.com/kumar-shridhar/PyTorch-Softplus-Normalization-Uncertainty-Estimation-Bayesian-CNN) in 2 days. 

However, we calculate the uncertainties on the test data on a given batch. So class wise uncertainty can only be derived on the validation set.
